arkouda/series.py

```diff
@@ -227,7 +225,13 @@ def locate(self, key):
 
             elif not isinstance(key0, pdarray):
                 # a list of scalers, convert into arkouda array
-                key = array(key)
+                try:
+                    val = array(key)
+                    if isinstance(val, pdarray):
+                        key = val
+                except Exception:
+                    raise TypeError(f"'key' parameter must be convertible to pdarray")
```

Comment:
You need to remove the `f` before the string. This is what is causing the CI to fail on `flake8`.